compiler/one-cmds/one-create-quant-dataset

```diff
@@ -0,0 +1,115 @@
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import h5py as h5
+import numpy as np
+import argparse
+import glob
+import os
+import onelib.utils as oneutils
+
+
+def get_parser():
+    """Create and return given the argument parser"""
+    parser = argparse.ArgumentParser(
+        description='command line tool to convert data files to hdf5 file')
+    parser.add_argument(
+        "-i",
+        "--input_data_format",
+        type=str,
+        help="Input data format of either rawdata or numpy",
+        choices=['rawdata', 'numpy'],
+        required=True)
+    parser.add_argument(
+        "-l",
+        "--data_list",
+        type=str,
+        help=
+        "Path to the text file where lists the absolute paths of data files to be converted.",
+        required=True)
+    parser.add_argument(
+        "-p", "--output_path", type=str, help="Path to the output file.", required=True)
+    return parser
+
+
+def create_hdf5(input_data_format, data_list, output_path):
+    """Create the hdf5 file using one of raw data file and numpy files listed in input_dir"""
+    h5_file = h5.File(output_path, 'w')
+    group = h5_file.create_group("value")
+
+    if input_data_format == 'rawdata':
+        # We assume the raw input data have the correct type/shape for the corresponding model
+        # If this flag is set in the hdf5 file, record-minmax will skip type/shape check
+        group.attrs['rawData'] = '1'
+
+    if not os.path.isfile(data_list):
+        raise FileNotFoundError("No such file. " + data_list)
+
+    num_converted = 0
+
+    # Data list
+    datalist = []
+    with open(data_list, 'r') as f:
+        lines = f.readlines()
+        for line in lines:
+            if line.strip():
+                filenames = line.rstrip().split(' ')
+                # A single input can indicate multiple files (multi-input)
+                for filename in filenames:
+                    if not os.path.isfile(filename):
+                        raise FileNotFoundError("No such file. " + filename)
+                datalist.append(filenames)
+
+    # Input files
+    for input_files in datalist:
+        sample = group.create_group(str(num_converted))
+        for idx, input_file in enumerate(input_files):
+            if input_data_format == 'rawdata':  # rawdata2hdf5
+                with open(input_file, 'rb') as f:
+                    raw_data = bytearray(f.read())
+                    sample.create_dataset(str(idx), data=raw_data)
+            else:  # numpy2hdf5
+                numpy_data = np.load(input_file)
```

Comment:
Can you make try-except around this function? If input_file is not loadable, it would be better to raise RuntimeError (f"{input_file} is not lodable by np.load. Please check it is a numpy file.")